test/integration/test_integration_zipkin.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 import pytest
 
-from integration_test_fixtures import (http_test_server_fixture)
+from integration_test_fixtures import (http_test_server_fixture, server_config)
```

Comment:
Doesn't look like `server_config` is used here?

Reply:
It doesn't look like, but `http_test_server_fixture` relies on it. If we don't explicitly import `server_config` here, we will get an error [1]. I think that is kind of silly, but it's also not
something I can do anything about, this seems to be related to something in the way that
pytest works. I've spend some time looking into this, and so far the easiest resolution seems to
be to just go with the flow and import the `server_config` fixture in scripts that use fixtures that rely on it. I'll add a comment about it for clarification.

[1]
```
  @pytest.fixture(params=determineIpVersionsFromEnvironment())
  def http_test_server_fixture(request, server_config):
E       fixture 'server_config' not found
>       available fixtures: capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, http_test_server_fixture, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, testrun_uid, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, worker_id
>       use 'pytest --fixtures [testpath]' for help on them.
```

Reply:
Ack, thanks for the explanation.